browser/brave_content_browser_client.cc

```diff
@@ -54,29 +54,18 @@ bool HandleURLRewrite(GURL* url,
     return true;
   }
 
-  if (url->SchemeIs(content::kChromeUIScheme) &&
```

Comment:
newtab handling here is no longer needed because of this:
https://github.com/brave/brave-core/pull/933/files#r234864961